### PR TITLE
Fixed malformed TensorFlow estimator declaration.

### PR DESCRIPTION
**Pull request by pccornel**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb [hunk 1]
```diff
@@ -137,11 +137,10 @@
     "\n",
     "estimator = TensorFlow(entry_point=training_script,\n",
     "                       role=role,\n",
-    "                       train_instance_count=1,\n",
-    "                       train_instance_type='ml.p2.xlarge',\n",
+    "                       instance_count=1,\n",
+    "                       instance_type='ml.p2.xlarge',\n",
     "                       framework_version=tf_version,\n",
-    "                       py_version='py3',\n",
-    "                       distributions={'parameter_server': {'enabled': True}})"
```
Comment (aaronmarkham) [inline aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb:144]: Makes sense that you don't need this with one node, but was this breaking something?
Can you put in the PR description which issue this fixes?

File: aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb [hunk 2]
```diff
@@ -137,11 +137,10 @@
     "\n",
     "estimator = TensorFlow(entry_point=training_script,\n",
     "                       role=role,\n",
-    "                       train_instance_count=1,\n",
-    "                       train_instance_type='ml.p2.xlarge',\n",
+    "                       instance_count=1,\n",
```
Comment (aaronmarkham) [inline aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb:140]: Is this a v1 --> v2 update?

File: aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb (hunk 1 above)
Comment (pccornel) [reply aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb:144]: I'm not aware of any specific issues in GitHub that this fixes.

File: aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb (hunk 2 above)
Comment (pccornel) [reply aws_sagemaker_studio/frameworks/tensorflow_mnist/tensorflow_mnist.ipynb:140]: Yes, this is a v1 --> v2 update.

**Review by aaronmarkham — APPROVED** [review]

**Pull request merged**